views-core/src/main/java/io/micronaut/views/model/security/SecurityViewModelProcessor.java

```diff
@@ -30,15 +30,15 @@
 import java.util.Optional;
 
 /**
- * Returns information about the current user so that it can be append it to the model being rendered.
+ * Returns information about the current user so that it can be appended it to the model being rendered.
  *
  * @author Sergio del Amo
  * @since 1.1.0
  */
 @Requires(property = SecurityViewModelProcessorConfigurationProperties.PREFIX + ".enabled", notEquals = StringUtils.FALSE)
 @Requires(beans = {SecurityFilter.class, SecurityService.class, SecurityViewModelProcessorConfiguration.class})
 @Singleton
-public class SecurityViewModelProcessor implements ViewModelProcessor {
+public class SecurityViewModelProcessor implements ViewModelProcessor<Map<String, Object>> {
```

Comment:
Updating existing processors to the new type is as simple as setting the type of Map<String, Object>